reference/5.1/Microsoft.PowerShell.Core/About/about_Arrays.md

```diff
@@ -689,11 +684,11 @@ $logs.Where({$_.CreationTime -gt $h}, 'Last', 5)
 #### SkipUntil
 
 The `SkipUntil` mode skips all objects in a collection until an object passes
-the script block expression filter. It then returns **ALL** remaining collection
-items without testing them. _Only one passing item is tested_.
+the script block expression filter. It then returns **ALL** remaining
+collection items without testing them. _Only one passing item is tested_.
 
 This means the returned collection contains both _passing_ and
-_non-passing_ items that have NOT been tested.
+_non-passing_ items that have _NOT_ been tested.
```

Comment:
Comment: This is a rare case where we're using casing and emphasis to highlight behavior. I think this is my first time seeing that in our docs.